### DISCUSSION TITLE
personal-development-mordad-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#19] [Announcements]
# personal-development-mordad-1402/

به سبک و سیاقی که شش ماهه در پیش گرفتم، گزارش توسعه فردی مرداد رو منتشر می‌کنم. برام جالب خواهد بود اگر بتونم نوشتن این گزارش‌ها رو چند سال ادامه بدم.
کتاب‌ ژن خودخواه اوایل این ماه بخشی از کتاب ژن‌ خودخواه ریچارد داوکینز رو خوندم و جذبش شدم. تو ترجمه‌های موجود، ترجمه دکتر جلال سلطانی رو خیلی پسندیدم و پنج فصل اول کتاب رو خوندم. می‌تونم بگم تا به حال یکی از قشنگ‌ترین کتاب‌هایی هست که خوندم.

https://aprd.ir/personal-development-mordad-1402/

**Comment by hiwasham** [comment]
امیر جان سرنخ های جالبی دادی دمت گرم
در مورد ورزش من هفت هشت ساله با علاقه یه چیزایی رو دنبال میکنم و اخیراً خیلی بیشتر شده
کلا چون فشار روانی و ... از سمت بیرون بیشتر شده، تلاش میکنم خودم بهتر عمل کنم و به اطرافیانم هم تجربه م رو بگم
با ابسیدین هم میدونی اینکار خیلی راحت تره. مثلا کل یادداشتهای این سالها از منابع خیلی خوب رو میشه سازماندهی و پابلیش کرد
منم مدتیه اینهارو تبدیل کردم به یه دوره ورزشی با تعدادی از بچه ها داریم انجامش میدیم
اگه مایل بودی بهم پیام بده 
ایمیلم داخل پروفایل متمم هست. ای دی تلگرامم هم همونه
با شناختی که ازت دارم حس میکنم خوشت میاد از این سیستم
قربانت

**Reply by masoudnasrollahi** [reply]
قضیه چیه؟ :thinking:

**Reply by pourmand1376** [reply]
یک دوره به نام «سبک زندگی شاداب (خواب، ورزش، تغذیه)» هست که من عضوش شدم و دارم پیگیری می‌کنم. دوره خوبی هست.

**Reply by masoudnasrollahi** [reply]
در موردش توی گزارش این ماه می‌خونیم. :)

**Comment by shahabson** [comment]
آزمون عملی موتور همون طور مثل قبله یا تغییر کرده؟

**Reply by pourmand1376** [reply]
سلام. 

من نمی‌دونم قبلاً چطور بوده. برای من این‌طور بود که ۴ عدد کله‌قندی با فاصله گذاشته بودند و کسی که می‌تونست بین کله‌قندی‌ها به خوبی عبور کنه، قبول می‌شد. 

برای ثبت‌نام هم می‌تونید به [سامانه آموزشگاه رانندگي ايرانيان](https://motor.nrdc.ir/P10/) مراجعه کنید. 

آزمون تئوری هم شفاهی بود و ۱۰ عدد تابلو بهمون نشون می‌دادند و اگر کسی می‌تونست کارکرد شش‌تاش رو توضیح بده، قبول بود.

**Comment by masoudnasrollahi** [comment]
* حس می‌کنم، فقط حس می‌کنیم که می‌شود جای دیگری هم پیاده کرد و نتیجه‌ی خوب گرفت. من تجربه‌ای ندارم. فکر می‌کنم با اینکه دوره‌هایی از این جنس تاثیرگذار هست و از ایده‌ها می‌شود استفاده کرد و کمک گرفت، اما برای گفت‌وگو با همسر نیاز است سراغ عناوین مربوط به خودش برویم. سراغ **جان گاتمن** و کتاب‌هایش. که البته آنجا دیگر خبری از محتوای شنیداری نیست و کسی برایش تولید محتوا بدین شکل نداشته است. (آقای هاشم علوی از گاتمن داخل گزارش‌هایشان نوشته بوده است)
+ ممنون که به درخواستم توجه کردید و برای *آموزش ساخت وب‌سایت شخصی* شروع به نوشتن کردید.
- **آمادگی جسمانی**؛ آمادگی جسمانی خود ورزش اجداد ماست و حتی بیشتر! سراغ آن را از آقای **فرهاد صنعت** بگیرید و حتما پی بگیرید. احتمالا پشیمان که نخواهید شد هیچ،‌ بدن هم می‌سازید. یک لحظه احساس هم پف داشتن نخواهید کرد. :smile:
